mariamihaelafainarea/assignment3/Assignment3Test

```diff
@@ -0,0 +1,226 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment3;
+
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+import jdk.nashorn.internal.ir.Assignment;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author maria
+ */
+public class Assignment3Test {
+    @Test
+    public void SimpleTest(){
```

Comment:
Tests are just methods annotated with @Test. Please follow the java convention of starting the method names with a small letter.

Reply:
okey.